TAF/utils/scripts/docker/get-compose-file.sh

```diff
@@ -18,30 +18,15 @@ mkdir temp
 if [ "$USE_RELEASE" = "nightly-build" ]; then
   # generate single file docker-compose.yml for target configuration without
   # default device services, i.e. no device-virtual service
-  ./sync-nightly-build.sh ${USE_SHA1} ${USE_NO_SECURITY} ${USE_ARM64}
-
-  # Need to remove the existing device services so the added ones below don't conflict
-  sed '/  device-rest:/,/- 127.0.0.1:49990:49990\/tcp/d' docker-compose-nexus${USE_NO_SECURITY}${USE_ARM64}.yml > temp/docker-compose-temp.yaml
-
-  # Insert device services into the compose file
-  sed -e '/app-service-rules:/r docker-compose-device-service.yaml' -e //N temp/docker-compose-temp.yaml > temp/device-service-temp.yaml
-
-  # Insert required services for end to end tests
-  sed -e '/app-service-rules:/r docker-compose-end-to-end.yaml' -e //N temp/device-service-temp.yaml > docker-compose.yaml
-
-  if [ "$USE_SECURITY" = '-security-' ]; then
-    # sed command of MacOS is in different syntax
-    if [ "$(uname)" = "Darwin" ]; then
-      sed -i '' "/hostname: edgex-vault-worker/i \\
-      \      ADD_SECRETSTORE_TOKENS: appservice-http-export-secrets
-      " docker-compose.yaml
-    else
-      sed -i "/hostname: edgex-vault-worker/i \\
-      \ADD_SECRETSTORE_TOKENS: appservice-http-export-secrets
-      " docker-compose.yaml
-    fi
-  fi
-
+  ./sync-nightly-build.sh "${USE_SHA1}" "${USE_NO_SECURITY}" "${USE_ARM64}" "-taf"
+  cp docker-compose-taf-nexus${USE_NO_SECURITY}${USE_ARM64}.yml docker-compose.yaml
+
+  sed -i '/PROFILE_VOLUME_PLACE_HOLDER: {}/d' docker-compose.yaml
+  sed -i 's/\CONF_DIR_PLACE_HOLDER/${CONF_DIR}/g' docker-compose.yaml
+  sed -i 's/\PROFILE_VOLUME_PLACE_HOLDER/${WORK_DIR}\/TAF\/config\/${PROFILE}/g' docker-compose.yaml
+  sed -i 's/\EXPORT_HOST_PLACE_HOLDER/${DOCKER_HOST_IP}/g' docker-compose.yaml
+  sed -i 's/\MQTT_BROKER_ADDRESS_PLACE_HOLDER/${MQTT_BROKER_IP}/g' docker-compose.yaml
+  sed -i 's/\LOGLEVEL: INFO/LOGLEVEL: DEBUG/g' docker-compose.yaml
```

Comment:
Would you please add a line to replace the service name `device-modbus:` to `edgex-device-modbus:`?
When we run the scalability test it used service name `edgex-device-modbus `instead of `device-modbus`.

Reply:
Done. Also updated the URL to point to master, so this PR is ready to go once we merge the developer-scripts PR. But first want to resolve tests issue with other PR and get it merged.